src/types/schemas/readme.md

```diff
@@ -0,0 +1,27 @@
+# refrence about molecule:
```

Comment:
Rename this file to `README.md` so that consistent with other repos in nervosnetwork org.